react-design/my-app/src/RotatingBanner.jsx

```diff
@@ -5,13 +5,24 @@ import PrevBtn from './PrevBtn';
 import Indicators from './Indicators';
 
 export default function RotatingBanner({ items }) {
-  const [currentIndex /*setCurrentIndex*/] = useState(0);
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  function handleSelect(index) {
+    setCurrentIndex(index);
+  }
+
   return (
     <div>
-      <Banner item={items[currentIndex]} />
-      <PrevBtn />
-      <Indicators count={items.length} current={currentIndex} />
-      <NextBtn />
+      <Banner items={items[currentIndex]} />
+      <div>
+        <PrevBtn items={items} current={currentIndex} set={setCurrentIndex} />
```

Comment:
Suggestion: I would name keep the prop name as `setCurrentIndex={setCurrentIndex}` so its clear in the child component that its the same `setCurrentIndex` function in the parent